config/flags.go

```diff
@@ -315,6 +316,10 @@ func addNodeFlags(fs *pflag.FlagSet) {
 	fs.Int(SnowMaxProcessingKey, snowball.DefaultParameters.MaxOutstandingItems, "Maximum number of processing items to be considered healthy")
 	fs.Duration(SnowMaxTimeProcessingKey, snowball.DefaultParameters.MaxItemProcessingTime, "Maximum amount of time an item should be processing and still be healthy")
 
+	// Simplex Consensus
+	fs.Duration(SimplexMaxProposalWaitKey, simplex.DefaultParameters.MaxProposalWait, "Max amount of time to wait before timing out on a round")
+	fs.Duration(SimplexMaxRebroadcastWaitKey, simplex.DefaultParameters.MaxRebroadcastWait, "Max amount of time to wait before rebroadcasting empty votes")
```

Comment:
> Max amount of time to wait before rebroadcasting empty votes

Perhaps something more user friendly like "Time to retry message transmission in case of network instability"